integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/merge/MergeTestCase.scala

```diff
@@ -723,6 +723,15 @@ class MergeTestCase extends QueryTest with BeforeAndAfterAll {
     assert(getDeleteDeltaFileCount("target", "0") == 0)
     checkAnswer(sql("select count(*) from target"), Seq(Row(3)))
     checkAnswer(sql("select * from target order by key"), Seq(Row("c", "200"), Row("d", "3"), Row("e", "100")))
+
+    // insert overwrite a partition. make sure the merge executed before still works.
+    sql(
+      """insert overwrite table target
+        | partition (value=3)
+        | select * from target where value = 100""".stripMargin)
+    checkAnswer(sql("select * from target"), Seq(Row("c", "200"), Row("e", "3"), Row("e", "100")))
```

Comment:
```suggestion
    checkAnswer(sql("select * from target order by key,value"), Seq(Row("c", "200"), Row("e", "100"), Row("e", "3")))
```

Reply:
I have modifed code according to your suggestion